Street Trees/STConfirmationPageViewController.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  STConfirmationPageViewController.swift
+//  Street Trees
+//
+//  Created by Pedro Trujillo on 10/17/16.
+//  Copyright © 2016 Code for Orlando. All rights reserved.
+//
+
+import UIKit
+import StreetTreesTransportKit
+
+
+//**********************************************************************************************************************
+// MARK: - Protocol
+
+protocol STConfirmationPageViewControllerDelegate: NSObjectProtocol {
+    func ConfirmationFormViewController(form: STConfirmationPageViewController, didCompleteWithWufooForm anWufooForm: STTKWufooForm)
+}
+
+//**********************************************************************************************************************
+
+class STConfirmationPageViewController: UIViewController, UITableViewDataSource, UITableViewDelegate
+{
+    
+    //var aTree:STTKStreetTree?
+    var anAddress: STTKStreetAddress?
+    var aContact: STTKContact?
+    
+    //just initial test values for vars
+    var treeName = "Example Tree"
+    var treeImage : UIImage?
+    var address = "101 Example Ave"
+    var cityStateZip = "Orlando, Fl 32801"
+    var name = "Bob Bobington"
+    var phone = "407-555-5550"
+    var email = "bob@google.com"
+    
+    weak var  delegate:STConfirmationPageViewControllerDelegate?
+    
+
+    override func viewDidLoad()
+    {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        
+        
+    // MARK: Get information from Wufoo Form:
+        
+        //treeName = (aTree?.name)!
+        //treeImage =  aTree.
+        address = (anAddress?.streetAddress)!  + " " +  (anAddress?.secondaryAddress)!
+        cityStateZip = "\(anAddress?.city), \(anAddress?.state) \(String((anAddress?.zipCode)!))"
+        name = (aContact?.name)!
+        phone = (aContact?.phoneNumber)!
+        email = (aContact?.email)!
+        
+    }
+
+    override func didReceiveMemoryWarning()
+    {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+
+    /*
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        // Get the new view controller using segue.destinationViewController.
+        // Pass the selected object to the new view controller.
+    }
+    */
+    
+    // MARK: - Table View Methods:
+    
+    func numberOfSectionsInTableView(tableView: UITableView) -> Int
+    {
+        return 1
+    }
+    
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int
+    {
+        return 8
+    }
+    
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell
+    {
+        let cell = tableView.dequeueReusableCellWithIdentifier("emptyCell", forIndexPath: indexPath)
```

Comment:
create a single cell in the tableview on the storyboard. looking at your code, a basic tableviewcell would be fine. Then you can query your datasource struct for the correct cell information.

I would ask the datasource for the enum at section and row and then create a switch for each enum variable instead of creating a ton of if statements. a large list of if statements can be rather cpu intensive and we don't want the cpu to do any additional work than what it has too.